packages/web-components/src/label/label.styles.ts

```diff
@@ -0,0 +1,50 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  colorNeutralForeground1,
+  colorNeutralForegroundDisabled,
+  colorPaletteRedForeground1,
+  fontFamilyBase,
+  fontSizeBase200,
+  fontSizeBase300,
+  fontSizeBase400,
+  fontWeightRegular,
+  fontWeightSemibold,
+  lineHeightBase200,
+  lineHeightBase300,
+  lineHeightBase400,
+} from '../theme/design-tokens.js';
+
+/** Label styles
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+
+  :host {
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    line-height: ${lineHeightBase300};
+    font-weight: ${fontWeightRegular};
+    color: ${colorNeutralForeground1};
+  }
+  .asterisk {
+    color: ${colorPaletteRedForeground1};
+  }
+  :host([size='small']) {
+    font-size: ${fontSizeBase200};
+    line-height: ${lineHeightBase200};
+  }
+  :host([size='large']) {
+    font-size: ${fontSizeBase400};
+    line-height: ${lineHeightBase400};
+    font-weight: ${fontWeightSemibold};
+  }
+  :host([weight='semibold']) {
+    font-weight: ${fontWeightSemibold};
+  }
```

Comment:
I'm just going to call this out - we're using JS here for things which can largely be set via pure CSS. It seems odd to me from a WC pattern to have an attribute which simply changes a single CSS property. I'm not going to block on this, but given the ability we have to use pure CSS we should really consider whether or not some of these first-class API's make sense.

Reply:
I mentioned this in previous PRs, never understood why we needed attributes for styling. Just use CSS on the component.

Reply:
Discussion here: https://github.com/microsoft/fluentui/pull/27362